### PR TITLE
Improve inline SVG detection

### PR DESCRIPTION
**Pull request by eltonmesquita**
Fix issue #1552

### CODE REVIEW

**Comment by patrickkettner** [comment]
@eltonmesquita - I thought you said you did _not_ want to run both checks?

**Comment by eltonmesquita** [comment]
Sorry, I've got a bit confused by your answer. But still, I think it is a better option, as it will not return any false positive, like the current test. If it is to be documented, where exactly this info should be? I don't see anything related to this kind of issue in the docs.

**Comment by patrickkettner** [comment]
we usually add a `knownBugs` field to the detect if there are known problems.

File: feature-detects/svg/inline.js [hunk 1]
```diff
@@ -7,8 +7,12 @@
   "notes": [{
     "name": "Test page",
     "href": "http://paulirish.com/demo/inline-svg"
+  }, {
+    "name: "Test page and results",
```
Comment (patrickkettner) [inline feature-detects/svg/inline.js]: you have a typo here that is breaking the parsing and thus the tests

`"name:` should be `"name":`

Comment (eltonmesquita) [inline feature-detects/svg/inline.js]: Sorry for that, ended up missing it. Fixed!

**Comment by patrickkettner** [comment]
Could you squish down everything to a single commit? then we can get this landed :D

**Comment by eltonmesquita** [comment]
Did I do it right, now? I don't know why the AppVeyour build failed, but it doesn't seem a problem with the code itself...

**Comment by patrickkettner** [comment]
appveyor is shakey - it messed up during the npm install. I kicked off the build again, just to double check.

**Comment by patrickkettner** [comment]
you would want to run `git rebase -i HEAD~7` and change all but the first `pick` to `f`s, then save and force push up the changes. That way the git history on Modernizr stays nice and clean

**Comment by eltonmesquita** [comment]
Did I get it now? Got take a course on git...

**Comment by patrickkettner** [comment]
@eltonmesquita seems like it got messed up :[

sorry again for complicated things. I cherrypicked and merged in via 60f6853567b4c2e47dd36007ee6a1a5de9304af5

thanks a ton for the PR!

**Comment by eltonmesquita** [comment]
No problem, this mess happened because I lacked some Git skills and time to review everything, it wasn't your fault. Next time I contribute, I'll do it in a clean way!
Thanks a lot for helping and discussing!

**Comment by patrickkettner** [comment]
:+1:

**Pull request closed without merging**